### DISCUSSION TITLE
Make `getting started` even more beginner friendly

### DISCUSSION THREAD

**Discussion by kpodsiad** [VirtusLab/scala-cli#593] [Ideas]
I wonder why https://scala-cli.virtuslab.org/docs/cookbooks/vscode isn't listed in the first paragraph in getting started or even in https://scala-cli.virtuslab.org/docs/getting_started#ide-support. `ide-support` links to the https://scala-cli.virtuslab.org/docs/guides/ide which also doesn't show how to setup Metals/IDE with scala-cli. 

What do you think about putting more emphasis on the steps described in https://scala-cli.virtuslab.org/docs/cookbooks/vscode ?

I've met at least a few people who was surprised that `scala-cli` is something more than command line tool and it can be used in vscode _with completions and other magic stuff_ without needing to create sbt projects for two files.

**Comment by bjornregnell** [comment]
Yes, using vscode without sbt is really great for beginners. I learned about `scala-cli setup-ide .` by chance as I happened to spot it in a Discord conversation. This is really useful. But as said by @kpodsiad it needs to be **discoverable**... Also, it would be nice if `code .` could "just work" without the `setup-ide .` in a dir with no build.sbt, as this might not be obvious to a beginner.
